### PR TITLE
Fix Issue #31: Remove network dependency from pkg_integrity tests

### PR DESCRIPTION
**Pull request by Gabib32**
This pull request uses two mock functions to imitate downloading files, and getting the status of a head request. This removes the network calls and allows tests to be ran in Travis CI.

### CODE REVIEW

File: tests/test_pkg_integrity.py [hunk 1]
```diff
@@ -1,50 +1,31 @@
 import os
+import shutil
+import mock
 import unittest
 import tempfile
 import pkg_integrity
 
+BASEDIR = os.getcwd()
+TESTDIR = BASEDIR + '/tests/testfiles/pkg_integrity'
```
Comment (bryteise) [inline tests/test_pkg_integrity.py]: os.path.join() is your friend (though we don't do that as much as we should in autospec).

**Comment by bryteise** [comment]
Looks good minus my nit. @Gabib32 is there a reason to not have these be all in one commit?

File: tests/test_pkg_integrity.py (hunk 1 above)
Comment (matthewrsj) [reply tests/test_pkg_integrity.py]: It's been getting better! Not all the way better, but better.

Comment (matthewrsj) [reply tests/test_pkg_integrity.py]: It doesn't look like BASEDIR gets used anywhere, is that the case? If so, can you just do 
```TESTDIR = os.path.join(os.getcwd(), 'tests/testfiles/pkg_integrity')```

**Comment by matthewrsj** [comment]
Other than the little thing +1 from me as well. Nice use of mock.patch.

**Comment by Gabib32** [comment]
Made those changes, and learned about squashing!

**Pull request merged**